docs/learn/learn-comparisons.md

```diff
@@ -1,63 +1,212 @@
+| **Scalability**              | Limited scalability as they are a single state machine and are only as scalable as the prover machine and computation requirement. Many zk-rollups disable cryptographic precompiles on mainnet as a result of the immense computational requirement for them. | High, with parallelization, but limited by gas costs on L1 chains like Ethereum.                                                                                             | Inherently scalable through native execution sharding and parachains operating in parallel. [Pipelining](./learn-async-backing.md) and [core scheduling](./learn-agile-coretime.md) increase throughput and scalability for the single rollup. Execution sharding is enabled by multiple virtual cores using [coretime](./learn-agile-coretime.md). |
+| **Decentralization**         | Centralization risks for sequencers and zk proof computation.                                                                                                                                                                                                  | Sequencer centralization risks; fewer validators can lead to potential censorship.                                                                                           | Collator decentralization is critical to ensure parachains remain trustless and independent.                                                                                                                                                                                                                                                        |
+| **Interoperability**         | Limited interoperability, often restricted to compatible L1s.                                                                                                                                                                                                  | Limited interoperability, often confined to the parent blockchain ecosystem. See the [comparison about interoperability](#interoperability-comparison) for more information. | Native interoperability through [XCM](./learn-xcm.md), allowing seamless communication between parachains having different logic. [Trustless bridges](./learn-bridges.md) can connect Polkadot to other blockchains.                                                                                                                                |
```

Comment:
Teams are working on native interoperability within rollup hubs, e.g. rollups within the OP Stack ecosystem will have "better" interop than independent L2s.
How exactly it's going to work is TBD, but at least being worked on by multiple teams